lca-cli/ibi-preparation/ibipreparation.go

```diff
@@ -153,8 +153,9 @@ func (i *IBIPrepare) diskPreparation() error {
 
 	i.cleanupDisk()
 
+	args := append([]string{"install", i.config.InstallationDisk}, i.config.CoreosInstallerParams...)
 	i.log.Info("Writing image to disk")
-	if _, err := i.ops.RunInHostNamespace("coreos-installer", "install", i.config.InstallationDisk); err != nil {
+	if _, err := i.ops.RunInHostNamespace("coreos-installer", args...); err != nil {
```

Comment:
why create args? why not?
```suggestion
	if _, err := i.ops.RunInHostNamespace("coreos-installer", "install", i.config.InstallationDisk, i.config.CoreosInstallerParams...); err != nil {
```